src/app/shared/components/header/header.component.ts

```diff
@@ -0,0 +1,39 @@
+import { Component, OnInit } from '@angular/core';
+import { FormControl } from '@angular/forms';
+import { Observable } from 'rxjs';
+import { startWith, map } from 'rxjs/operators';
+@Component({
+    selector: 'app-header',
+    templateUrl: './header.component.html',
+    styleUrls: ['./header.component.scss'],
+})
+export class HeaderComponent implements OnInit {
+    control = new FormControl();
+    streets: string[] = [
```

Comment:
don't cast type here
Ts is rather smart to understand that this is array of strings